packages/react-native-editor/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ For each user feature we should also add a importance categorization label  to i
 -->
 
 ## Unreleased
+
+## 1.97.0
 -   [*] [internal] Upgrade compile and target sdk version to Android API 33 [#50731]
```

Comment:
It may be worth including an entry for the disabled cell icon as well. 

```suggestion
-   [*] [internal] Upgrade compile and target sdk version to Android API 33 [#50731]
-   [*] Display lock icon in disabled state of `Cell` component [#50907]
```